### PR TITLE
docs: add note about enabling the Plugin

### LINKED ISSUE #1564: [Docs] Add how to enable Plugins in the ApisixRoute#Plugins doc
The current guide mentions how you can configure Plugins in ApisixRoute resource but does not mention how to enable the Plugins that are not enabled in APISIX by default. A note on updating the values.yaml file during Helm install of APISIX could be mentioned as this is causing confusion: https://the-asf.slack.com/archives/CUC5MN17A/p1672147171239759

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat <navendu@apache.org>

Fixes #1564 

Add a note mentioning how to enable the Plugins that are not enabled by default.

### CODE REVIEW

File: docs/en/latest/concepts/apisix_route.md [hunk 1]
```diff
@@ -178,6 +178,12 @@ spec:
 
 APISIX's [80+ Plugins](https://apisix.apache.org/docs/apisix/plugins/batch-requests/) can be used with APISIX Ingress. These Plugins have the same name as in the APISIX documentation.
 
+:::note
+
+If the Plugin is not enabled in APISIX by default, you can enable it by adding it to the `plugins` attribute in your `values.yaml` file while installing APISIX and Ingress controller via Helm.
```
Comment (tao12345666333) [inline docs/en/latest/concepts/apisix_route.md:183]: I think we can also remind users to directly modify APISIX configuration files, what do you think?

Comment (navendu-pottekkat) [reply docs/en/latest/concepts/apisix_route.md:183]: Sure. We can add the line "You can also modify your APISIX configuration file directly (`conf/config.yaml`)."

Comment (tao12345666333) [reply docs/en/latest/concepts/apisix_route.md:183]: great!

Comment (navendu-pottekkat) [reply docs/en/latest/concepts/apisix_route.md]: Updated.

Comment (tao12345666333) [reply docs/en/latest/concepts/apisix_route.md]: Or if it is deployed in Kubernetes, it can be modified through configmap

**Review by tao12345666333 — APPROVED** [review]

File: docs/en/latest/concepts/apisix_route.md (hunk 1 above)
Comment (navendu-pottekkat) [reply docs/en/latest/concepts/apisix_route.md]: Adding through ConfigMap would mean we are indirectly updating the `config.yaml` file right? Would it be confusing to the user if we mention it in a sentence? There are tutorials that mentions how a user can do that. We can mention these two methods as the easiest and the recommended ways to do that. A user can choose to go the ConfigMap path if they want. What do you think?

Comment (tao12345666333) [reply docs/en/latest/concepts/apisix_route.md]: ok, that make sense

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**